src/sync-accumulator.ts

```diff
@@ -43,6 +44,12 @@ export interface IMinimalEvent {
     unsigned?: IUnsigned;
 }
 
+export interface AccumulatedReceipt {
```

Comment:
Looks like this belongs to the `ReceiptAccumulator` and should eventually be moved to the `receipt-accumulator` module?

Reply:
Fixed in next PR: https://github.com/matrix-org/matrix-js-sdk/pull/3320